app/modules/munkireportinfo/munkireportinfo_model.php

```diff
@@ -56,7 +56,6 @@ public function process($data)
             // Convert version to int
             if (isset($plist['version'])) {
                 $digits = explode('.', $plist['version']);
-                array_pop($digits);
```

Comment:
It was a bug discovered with the MRinfo module. Wasn't supposed to be in this PR. Will remove change and make PR for it.